services/dynamic-sidecar/src/simcore_service_dynamic_sidecar/core/application.py

```diff
@@ -36,12 +37,61 @@
 S     S    S      ss.  S S     S S sSSs  S       S sSSO S sS'
 S     P    S         b S S     P S       S       S    O S   S
 S    S     S         P S S    S  S        S      S    O S    S
-P ss"      P    ` ss'  P P ss"   P sSSss   "sss' P    P P    P   {}
+P ss"      P    ` ss'  P P ss"   P sSSss   "sss' P    P P    P   {} 🚀
 
 """.format(
     f"v{__version__}"
 )
 
+TSCHUESS_MSG = "{:=^100}".format("🎉 App shutdown completed 🎉")
```

Comment:
While I enjoy the pun, please default to English.

Reply:
I am referring to `TSCHUESS_MSG`

Reply:
heheh ok